app/Models/Room.php

```diff
@@ -215,4 +215,16 @@ private function notWithinAvailabilities($startDate, $endDate)
       $endTime > $availabilityEnd->closing_hours;
   }
 
+ 
+
+  public function minimunReservationTime($startDate, $endDate, $fail)
+  {
+    $startTime = Carbon::parse($startDate);
+    $endTime = Carbon::parse($endDate);
+    if($endTime->diffInMinutes($startTime) < 30){
+       $fail('The room cannot be booked for less than 30 min');
+    }
+  }
+
+
```

Comment:
should follow https://www.php-fig.org/psr/psr-2/ by using 4 spaces instead of 2.

Reply:
also theres a typo in minimum